app/enterprise/2.2.x/deployment/kong-user.md

```diff
@@ -43,3 +46,19 @@ configuration property.
     ```sh
     kong start
     ```
+
+## Run {{site.ee_product_name}} as a custom non-root user
+
+It is also possible running Kong as a custom non-root user. Since all the files installed by the {{site.ee_product_name}} package are owned by the `kong` group, a user that belongs to that group should be permitted to perform the same operations as the `kong` user.
```

Comment:
```suggestion
It is also possible to run Kong as a custom non-root user. Since all the files installed by the {{site.ee_product_name}} package are owned by the `kong` group, a user that belongs to that group should be permitted to perform the same operations as the `kong` user.
```